Darwin/StitchCore/StitchCore/Core/Auth/StitchUser.swift

```diff
@@ -62,6 +62,17 @@ public protocol StitchUser: CoreStitchUser {
      */
     var lastAuthActivity: TimeInterval { get }
 
+    /**
+     You can store arbitrary data about your application users
+     in a MongoDB collection and configure Stitch to automatically
+     expose each user’s data in a field of their user object.
+     For example, you might store a user’s preferred language,
+     date of birth, or their local timezone.
+
+     If this value has not been configured, it will be empty.
```

Comment:
[nit] same here about `functionality` vs `value`

Reply:
I actually copied this from the docs website, but I'll change it here.